src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.tsx

```diff
@@ -139,7 +139,7 @@ class OrganizationDeveloperSettings extends AsyncView<Props, State> {
     return (
       <div>
         <SettingsPageHeader title={t('Developer Settings')} />
-        <AlertLink href="https://docs.sentry.io/workflow/integrations/integration-platform/">
```

Comment:
I think we need to keep the `href` prop for the same reasoning discussed earlier - we need to be sure that `href` and `to` props are two very distinct things. People need to be aware of the differences between the two and having them combined like this can be confusing.